src/Sylius/Bundle/AssociationBundle/composer.json

```diff
@@ -0,0 +1,60 @@
+{
+    "name": "sylius/association-bundle",
+    "type": "symfony-bundle",
+    "description": "Products association for your Symfony2 applications.",
+    "keywords": ["shop", "ecommerce", "products", "product", "assortment", "association"],
+    "homepage": "http://sylius.org",
+    "license": "MIT",
+    "authors": [
+        {
+          "name": "Paweł Jędrzejewski",
```

Comment:
First three authors are idented with 2 spaces while there is used alignment of 4 spaces everywhere else.